### PR TITLE
Update install-from-packages.md

### PR DESCRIPTION
**Pull request by mikejmets**
Remove yarn.lock can assist changing the yarn version

## First-time contributors

You **must** read and follow our [First-time contributors](https://6.docs.plone.org/contributing/first-time.html).

---

## Submit a pull request

Thank you for your contribution to the Plone Documentation.

Before submitting this pull request, please make sure you follow our guides:

-   [Contributing to Plone documentation](https://6.docs.plone.org/contributing/index.html)
-   [Building and checking the quality of documentation](https://6.docs.plone.org/contributing/setup-build.html)
-   [Authors guide](https://6.docs.plone.org/contributing/authors.html)
-   [MyST reference](https://6.docs.plone.org/contributing/myst-reference.html)

## Issue number

- Fixes #

## Description

_Write a description of the fixes or improvements._

### CODE REVIEW

File: docs/install/install-from-packages.md [hunk 1]
```diff
@@ -184,6 +184,14 @@ Install the latest Yarn 3 version (not the Classic 1.x one) using `npm`.
     yarn set version 3.x
     ```
 
+    If the version doesn't change you can try deleting the yarn.lock file, setting the version and installing again.
```
Comment (stevepiercy) [inline docs/install/install-from-packages.md]:
```suggestion
    If the version doesn't change, you can try deleting the {file}`yarn.lock` file, setting the version, and installing again.

```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thank you! This has bitten me in the past.

Minor grammar and MyST enhancement, and it's good to merge.

**Comment by stevepiercy** [comment]
The one failed CI job is due to a bad cert on pastanaga.io. I've notified the site owners. We can ignore this one failure.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@mikejmets is this still a draft, or ready to merge?

**Comment by mikejmets** [comment]
It can go but I dunno how to lift the Draft status :-)

**Pull request merged**